resources/utility-scripts/wait_for_startup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+DB_NUMBER=9
```

Comment:
The value affected to this variable should be documented somewhere, so that we know we need to update it if the number of containers running in a deployment changes.

Reply:
And maybe it should also be defined in the .travis.yml file (if possible).